websocket/integrationTests/clientAndServer_test.go

```diff
@@ -163,3 +165,95 @@ func TestStartServerStartClientCloseServer(t *testing.T) {
 	require.Equal(t, 200, numMessagesReceived)
 	require.Equal(t, sentMessages, receivedMessages)
 }
+
+func TestStartServerStartClientAndSendABigMessage(t *testing.T) {
+	url := "localhost:8833"
```

Comment:
we need to figure out how to dynamically change this port.

Reply:
refactored.